app/jobs/application_submission_job.rb

```diff
@@ -4,6 +4,8 @@ class ApplicationSubmissionJob < ApplicationJob
   queue_as :default
 
   def perform(user:, email_template:)
+    return if Feature.ecf_api_disabled?
+
     if user.synced_to_ecf?
       update_ecf_user_details(user:)
     else
```

Comment:
not implemented in this pr but it's interesting that we call all applications on a user to send emails/create applications in ECF for, wonder if this is the cause of the duplication we see in ECF, or if there is a reason we are doing this for all applications rather than the new one created